src/lib/containers/QueryWrapperMenu.tsx

```diff
@@ -235,8 +240,9 @@ export default class QueryWrapperMenu extends React.Component<
     } = this.props
 
     let sql = ''
+    let selectedFacets = undefined
     if (menuConfig) {
-      sql = this.getCurrentSqlFromActiveTab()
+      ;({ sql, selectedFacets } = this.getCurrentSqlFromActiveTab())
```

Comment:
It's a bit awkward that eslint puts the semi-colon on the left hand side instead of the right hand side, but its there to allow for the sql and selected facets to be assigned from the destructured statement, mozilla has a better explanation here under 'Assignment without declaration'
  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring